proto/one/historian/ingest/file_configuration.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+message FileConfiguration
+{
+     /// Where to monitor for new files
+     string MonitorPath = 1;
```

Comment:
Does the casing matter for these? I noticed all the other `message` classes use camel case for properties, while these are Pascal case.

Reply:
thanks for catching this.  not sure if the casing matters, but i should fix this.